api/v1/sriovnetwork_types.go

```diff
@@ -60,6 +60,13 @@ type SriovNetworkSpec struct {
 	// MetaPluginsConfig configuration to be used in order to chain metaplugins to the sriov interface returned
 	// by the operator.
 	MetaPluginsConfig string `json:"metaPlugins,omitempty"`
+	// LogLevel sets the log level of the SRIOV device plugin - either of panic, error, warning, info, debug. Defaults
```

Comment:
What do you think about setting the default explicitly to info using +kubebuilder:default:= "info".

This way, the parameter will always be populated and users will clearly see what log level they are in and where to change it. (I know they can read the docs to find it out, but this improves user experience in my opinion).